src/megaclient.cpp

```diff
@@ -10117,9 +10179,21 @@ bool MegaClient::startxfer(direction_t d, File* f, bool skipdupes)
                 t->size = f->size;
             }
 
+            t->lastaccesstime = time(NULL);
```

Comment:
Despite we talked in the office, I have checked it and `lastaccesstime` is an absolute time, so that code is correct. That absolute time is used to control the expiration of temporary URLs, so it's needed that it's valid for different executions of the app.